### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by StephenJWaite**
Merging the develop branch into the master to test pull request notifications

### CODE REVIEW

File: tests/test_stephen_addition.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#a test function for my addition
+import unittest
+import sys
```
Comment (StephensGitTest) [inline tests/test_stephen_addition.py:3]: Oh uou can leave line comments thats cool

File: tests/test_stephen_devision.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# a test function for my addition
+import unittest
+import sys
+
+sys.path.insert(0, "./src/")
+import PythonScript as PS
```
Comment (StephensGitTest) [inline tests/test_stephen_devision.py:6]: dfdfdf

Comment (StephensGitTest) [reply tests/test_stephen_devision.py:6]: not sure what I have done here

**Review by StephensGitTest — COMMENTED** [review]
Creat enew yaml file

Python Script had a variable d that is unused

File: tests/test_stephen_addition.py (hunk 1 above)
Comment (StephenJWaite) [reply tests/test_stephen_addition.py:3]: Okay good job, i will resolve this

**Comment by StephenJWaite** [comment]
What a great pull request

**Pull request merged**